score/mw/com/impl/configuration/config_parser.cpp

```diff
@@ -764,15 +926,28 @@ auto ParseLolaEventTypeDeployments(const score::json::Any& json, LolaServiceType
 // coverity[autosar_cpp14_a15_5_3_violation]
 auto ParseLolaFieldTypeDeployments(const score::json::Any& json, LolaServiceTypeDeployment& service) noexcept -> bool
 {
-    const auto& fields = json.As<score::json::Object>().value().get().find(kFieldsKey.data());
-    if (fields == json.As<score::json::Object>().value().get().cend())
+    auto json_obj = json.As<score::json::Object>();
+    if (!json_obj.has_value()) {
+        return false;
```

Comment:
Behavioral change. Should terminate.

```suggestion
        score::mw::log::LogFatal("lola") << "LolaFieldTypeDeployments not parsable.";
        SCORE_LANGUAGE_FUTURECPP_ASSERT_PRD(false);
```